_includes/guides/create-dockerfile.md

```diff
@@ -0,0 +1,39 @@
+A Dockerfile is a text document that contains the instructions to assemble a
```

Comment:
Oh, perhaps worth mentioning; I tried to distinguish "Dockerfile" (the "concept") from `Dockerfile` (the _file_), and only formatting  `Dockerfile` as `code` if it's referring to the _file_. It's tricky to make that distinction everywhere though.